src/views/forms/FormTransferTransaction/FormTransferTransactionTs.ts

```diff
@@ -703,9 +700,19 @@ export class FormTransferTransactionTs extends FormTransactionBase {
      * @param {number} maxFee
      */
     private createTransactionCommandForFee(maxFee: number): TransactionCommand {
+        const mosaicsInfo = this.$store.getters['mosaic/mosaics'] as MosaicModel[];
         const transactions = this.getTransactions().map((t) => {
             //@ts-ignore
             t.maxFee = UInt64.fromUint(maxFee);
+
+            if (t.mosaics.length) {
+                t.mosaics.map((m) => {
+                    const info = mosaicsInfo.find((i) => i.mosaicIdHex === m.id.toHex());
+                    const div = info ? info.divisibility : 0;
+                    const amount = m.amount.compact() / Math.pow(10, div);
+                    return new Mosaic(m.id, UInt64.fromUint(amount));
+                });
```

Comment:
What is this good for? The result of that map is not used...

Reply:
Refactored.